### PR TITLE
Add tests that set up a nonempty returndata buffer and then call something that reverts

### PR DESCRIPTION
**Pull request by pirapira**
This should fulfill: non-empty returndata buffer --CALL --> REVERT <-- RETURNDATASIZE, to detect "not setting returndata buffer after REVERT"

### CODE REVIEW

File: src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+		    "b0005374fce5edbc8e2a8697c15331677e6ebf0b" : {
+			"nonce" : "0",
+			"storage" : {
+                            "0x02" : "0x01"
+                        }
+		     }
+                }
+            },
+	    {
+		"comment" : "OOG",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json]: comment fields should start with "//comment" :

File: src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	     "094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json]: this address is a dublicate of previous one

File: src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+	    "b3305374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ (CALL 0 0xffff5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) [[4]] (CALL 50000 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) [[5]] (RETURNDATASIZE) }",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "c94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ [[1]] 12 (REVERT 0 1) [[3]] 13 }",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json]: could you put more comments to describe whats happening in the calls?

File: src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "ffff5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ [1] 12 (RETURN 0 64) }",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stRevertTest/RevertOpcodeInCallsOnNonEmptyReturnDataFiller.json]: could you put more comments to describe whats happening in the calls?

**Comment by pirapira** [comment]
* Updated the filler according to the comment.
* Refilled using the new `testeth --filltests --fillchain`.

**Pull request merged**